<ui version="4.0" >
 <class>TransferCurve</class>
 <widget class="QWidget" name="TransferCurve" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Channel</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>ColorSelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ColorSelector" >
       <item>
        <property name="text" >
         <string>Value</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Blue</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="1" >
      <widget class="QwtPlot" name="CurveDisplay" >
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="verticalScale" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Ignored" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="horizontalScale" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="LoadButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >qwt_plot.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
